purchase_advance_payment/tests/test_purchase_advance_payment.py

```diff
@@ -46,6 +46,21 @@ def setUpClass(cls):
         cls.purchase_order_1 = cls.env["purchase.order"].create(
             {"partner_id": cls.res_partner_1.id}
         )
+        cls.purchase_order_2 = cls.env["purchase.order"].create(
```

Comment:
can you please add this after the call to super above? 
```
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
``

Reply:
I just update the branch, please check again